### PR TITLE
SW-2883 Error page copy is missing

### PR DESCRIPTION
**Pull request by constanzauanini**
The error page was not showing the labels, this was because locale was not set for the error page.
Adding AppBootstrap to error page to allow setting the locale

### CODE REVIEW

**Review by charlie-tf — COMMENTED** [review]
Can these layers be placed inside `AppError` similar to the way they are layered in `App`? To avoid importing those components to `index`...

**Comment by constanzauanini** [comment]
> Can these layers be placed inside `AppError` similar to the way they are layered in `App`? To avoid importing those components to `index`...

That's a good suggestion!

**Review by charlie-tf — APPROVED** [review]
👍

File: src/AppError.tsx [hunk 1]
```diff
@@ -3,17 +3,24 @@ import { Route, Switch } from 'react-router-dom';
...
-    <Switch>
-      <Route exact path={APP_PATHS.ERROR_FAILED_TO_FETCH_ORG_DATA}>
-        <ErrorContent />
-      </Route>
-      <Route path={APP_PATHS.ERROR}>
-        <QueryParamsError />
-      </Route>
-    </Switch>
+    <AppBootstrap>
+      <Provider store={store}>
```
Comment (karthikbtf) [inline src/AppError.tsx:13]: what is the store for? app version detector?

Comment (constanzauanini) [reply src/AppError.tsx:13]: I don't think we are using the store here, just wanted to have same initialization as App, but probably we can remove it

Comment (karthikbtf) [reply src/AppError.tsx:13]: no worries, was just curious.. we can leave it if we want same initialization / consistency

**Pull request merged**